simapp/v2/app_test.go

```diff
@@ -21,6 +22,7 @@ import (
 	serverv2 "cosmossdk.io/server/v2"
 	comettypes "cosmossdk.io/server/v2/cometbft/types"
 	"cosmossdk.io/store/v2/db"
+	authtypes "cosmossdk.io/x/auth/types"
```

Comment:
?

Reply:
probably from some of my tests, will remove